src/services/me/docs/openapi.yaml

```diff
@@ -12,57 +12,177 @@ info:
+          properties:
+            form:
+              type: string
+            source:
+              type: string
+            dateOfPrivacyConsent:
+              type: string
+              format: date-time
+            privacyConsent:
+              type: boolean
+            dateOfTermsOfUseConsent:
+              type: string
+              format: date-time
+            termsOfUseConsent:
+              type: boolean
+        parentConsents:
+          type: array
+          items:
+            type: object
+            properties:
+              form:
+                type: string
+              source:
+                type: string
+              dateOfPrivacyConsent:
+                type: string
+                format: date-time
+              privacyConsent:
+                type: boolean
+              dateOfTermsOfUseConsent:
+                type: string
+                format: date-time
+              termsOfUseConsent:
+                type: boolean
+    role:
+      description: A user role Object
+      type: object
+      properties:
+        _id:
+          type: string
+          pattern: '^[0-9a-f]{24}$'
+        name:
+          type: string
+        updatedAt:
+          type: string
+          format: date-time
+        createdAt:
+          type: string
+          format: date-time
+        roles:
+          type: array
+          description: Array of roles this role is inheriting from
+          items:
+            type: string
+            pattern: '^[0-9a-f]{24}$'
+        permissions:
+          type: array
+          items:
+            type: string
+        __v:
+          type: integer
+        displayName:
+          type: string
+        id:
+          type: string
+          pattern: '^[0-9a-f]{24}$'  
+    user_full:
+      description: Data Transfer Object containing fully populated userdata
+      type: object
+      required:
+        - firstName
+        - lastName
+        - email
+      properties:
+        _id:
+          type: string
+          pattern: '^[0-9a-f]{24}$'
+        __v:
+          type: integer
+        firstName:
+          type: string
+        lastName:
+          type: string
+        email:
+          type: string
+          format: 'email'
+        updatedAt:
+          type: string
+          format: date-time
+        createdAt:
+          type: string
+          format: date-time
+        birthday:
+          type: string
+          format: date-time
+        preferences:
+          type: object
+          properties:
+            firstLogin:
+              type: boolean
+          additionalProperties: true
+        schoolId:
+          type: string
+          pattern: '^[0-9a-f]{24}$'
+        roles:
+          type: array
+          items:
+            $ref: '#/components/schemas/role'
+        consent:
+          $ref: '#/components/schemas/consent'
+        forcePasswordChange:
+          type: boolean
+        fullName:
+          type: string
+        id:
```

Comment:
Do we really have both `_id` and `id`?

Reply:
yes :)

Reply:
yes, we currently do.
Its a bit unnecessary, mongoose adds the id field as a convinience to access the string representation of the id, but to the outside we always pass the _id field as string too.
But honestly I would consider that out of scope

Reply:
Thanks for the explanation. Should indeed be out of scope of this PR.